AElf.Kernel.Core/Blockchain/Application/ITransactionResultService.cs

```diff
@@ -53,6 +53,9 @@ public async Task<TransactionResult> GetTransactionResultAsync(Hash transactionI
 
             var chain = await _blockchainService.GetChainAsync();
             var hash = chain.BestChainHash;
+            var until = chain.LastIrreversibleBlockHeight > ChainConsts.GenesisBlockHeight
```

Comment:
make chain.LastIrreversibleBlockHeight default value is ChainConsts.GenesisBlockHeight

Reply:
ok

Reply:
do not need check here